WalletWasabi.Fluent/Extensions/FeeDisplayUnitExtension.cs

```diff
@@ -0,0 +1,15 @@
+using NBitcoin;
+using WalletWasabi.Fluent.Models;
+
+namespace WalletWasabi.Fluent.Extensions;
+
+public static class FeeDisplayUnitExtension
+{
+	public static MoneyUnit ToMoneyUnit(this FeeDisplayUnit feeDisplayUnit) =>
```

Comment:
@turbolay 
It should have gone to `CurrencyExtensions.cs` instead of a separate file.